### PR TITLE
Update README.md with the new seed db file names

### PR DESCRIPTION
**Pull request by oplantalech**
This PR should be merged after PR cbioportal/datahub#21 from datahub is accepted, to keep the documentation consistent.

### CODE REVIEW

**Comment by fedde-s** [comment]
Looks good, I'll merge it once I notice the dependency has been met. @oplantalech, do you want to squash the commits before I do? Looks like the second one is mostly just a fixup of the first.

**Comment by oplantalech** [comment]
@fedde-s Squashed.

File: README.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ docker run -d --restart=always \
   -e MYSQL_DATABASE=cbioportal \
   -v /<path_to_save_mysql_db>/db_files/:/var/lib/mysql/ \
   -v /<path_to_seed_database>/cgds.sql:/docker-entrypoint-initdb.d/cgds.sql:ro \
-  -v /<path_to_seed_database>/seed-cbioportal_no-pdb_hg19.sql.gz:/docker-entrypoint-initdb.d/seed_part1.sql.gz:ro \
-  -v /<path_to_seed_database>/seed-cbioportal_only-pdb.sql.gz:/docker-entrypoint-initdb.d/seed_part2.sql.gz:ro \
+  -v /<path_to_seed_database>/seed-cbioportal_<genome_name>_<seed_version>.sql.gz:/docker-entrypoint-initdb.d/seed_part1.sql.gz:ro \
```
Comment (pieterlukasse) [inline README.md]: can we call this `<genome_build>` instead?

Comment (fedde-s) [reply README.md]: Makes sense: that would be more consistent with the usage of the word in other places such as https://github.com/cBioPortal/cbioportal/blob/v1.5.1/docs/Using-the-dataset-validator.md#validation-of-non-human-data

**Comment by oplantalech** [comment]
@fedde-s @pieterlukasse I've changed `<genome_name>` to `<genome_build>`.

**Review by pieterlukasse — APPROVED** [review]

**Pull request merged**